Hello Microverse Pull Request

PraveenKumar141095 opened the pull request:
A new branch is created. Features:
-Added an HTML file with website content.
-Added a CSS file that applies the style to the HTML file.
-Added a .gitignore file to filter unwanted files.
-Added a Linter file to check possible bugs on the files.

Fixed:

Code errors on the CSS file.
Pull Request description.
Readme.md (main branch) Getting started content.

PraveenKumar141095 committed:
Add html file

PraveenKumar141095 committed:
Add css file

PraveenKumar141095 committed:
style.css added

PraveenKumar141095 committed:
added css

levy002 reviewed (changes requested):
## Hello @Praveen1995-10-14   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)

## Highlights
- Linters are passing :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to add a good PR description. It makes your repo looks more professional. :wink: 

   - Example of good PR
![image](https://user-images.githubusercontent.com/87197412/184990532-da778ab4-c789-4136-90a9-629eedd969da.png)

- -  [ ] Kindly, make sure to flow the correct Github flow as it is the only way we use to ask for a code review by creating a pull creating to compare `feature` and our main branch. check the process:
   - The GitHub flow means that every time we want to implement a new feature, instead of going ahead and pushing everything to `main`.
   - First of all, we create the new `feature branch` as you did and push every change we want to make.
   - After that, we create a pull request and ask for a review (that is the professional way). 
   -  :stop_sign: :negative_squared_cross_mark: but we don't directly merge the `feature branch` into the `main` branch without receiving feedback from our code reviewer. We don't even push anything directly to the main branch.
   - We submit our PR link via the dashboard and wait for code reviewer feedback.
   - If the code reviewer requested some changes, we have to address those requested changes and submit again the PR link via the dashboard.
   - If our PR is approved, we have first to check if there is [OPTIONAL] requirement and address them. After all of that, we can go ahead and **MERGE**  our PR into the `main` branch.
   
 - [x] :heart_decoration: Kindly, go ahead and implement the correct GitHub flow as it is a project requirement. Let's make our work looks more professional :muscle:  
     - Add the `.gitignore` file into this `feature branch`, not the `main` branch
     - Add a descriptive README file into this branch, not in the `main` branch
     - I recommend you to take a step back and read this[ lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) again to understand it clearly.
   
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Kindly make sure to write your commit message in imperative tense and capitalized it. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

PraveenKumar141095 committed:
update ReadMe with basic project information.

bobb-Rob commented on README.md:
```diff
@@ -1 +1,38 @@
+5-->Write short, detailed and easy to understand descriptions of pull requests.
+6-->Demonstrate an ability to submit a project for code review.
+
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+
+
+## Authors
+
```
- [ ] As pointed out by the previous reviewer, it is a requirement for this project, and also in future projects in microverse, endeavor to add it. 👉 Kindly make sure to add a good Pull Request description. This description is a brief outline of things you implemented in that pull request. 😉

- [ ] Good job adding this README file as requested by the previous reviewer, your repository is now professional. But there is a missing section, the "Getting Started" section. This section is important in giving intending users of your repo a proper guide on how to use it. Kindly add this section but have non-technical users (like HR persons) in mind.

bobb-Rob reviewed (changes requested):
## STATUS: CHANGES REQUIRED ♻️
Hi @Praveen1995-10-14 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>
![](https://camo.githubusercontent.com/34482800e1b92cb79ddca06edb4051edc407cb190f896e1e949e596aa9073851/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)

## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

PraveenKumar141095 committed:
Update README.md

i added project description ,acknowledgments, contributing section

PraveenKumar141095 committed:
Merge branch 'main' into htmlCss-branch

PraveenKumar141095 committed:
Update README.md files

BertrandConxy commented on README.md:
```diff
@@ -1,15 +1,22 @@
-5-->Write short, detailed and easy to understand descriptions of pull requests.
-6-->Demonstrate an ability to submit a project for code review.
+
+>The main goals of this project are:
+ - How to create branches with different base branches.
+ - To deal with version control.
+ - Follow GitHub flow.
+ - Open pull request.
+ - Write descriptive to understand commit messages.
+ - Write short, detailed and easy to understand descriptions of pull requests.
+ - Demonstrate an ability to submit a project for code review.
+# In this project,3 files are added,
+
+  - A .html file that includes a header with the text "Hello Microverse!".
+
+  - A .css file that includes one class with styles for your header.
+
+  - A test.md file with the text "This file should be ignored by git".
```
- [ ] Great job so far 👏. However, as the previous reviewer mentioned, in order for your README file to look professional, you need to add the `Getting started` section where you describe the procedures to clone and use your repository locally. T**his helps your fellow developers to know how they can use your project and this is great for development 😃 so please replace these highlighted texts and add the Getting started section**

PraveenKumar141095 replied:
I mentioned everything:-
on headings, i just mentioned what is the goal of the projects and after that, I wrote what is in the projects.

PraveenKumar141095 replied:
so i have to remove this line: In this project,3 files are added.?

PraveenKumar141095 replied:
I have just the last review option left plz tell me where I am going wrong.

BertrandConxy reviewed (changes requested):
Hi @Praveen1995-10-14 , 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 💪 
- No linter errors
- correct flow is used
- good improvement based on the previous reviewer's suggestions



 ## Required Changes ♻️

- [ ] As mentioned by previous reviewer, it is important to add brief description of what changes you have introduces in the branch you are working on so that anyone who is going to review your code will know what to review very easily. **HINT** It looks much better when they are listed .
For example :
```
- Add HTML file
- Add CSS file
- Set up linters
```
Please edit your PR description to include a good description as suggested above 🙏 

_Check other comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

PraveenKumar141095 committed:
update README.md file

PraveenKumar141095 committed:
update ReadMe.md

cynthiainga commented on README.md:
```diff
@@ -1,15 +1,23 @@
+ - Write descriptive to understand commit messages.
+ - Write short, detailed and easy to understand descriptions of pull requests.
+ - Demonstrate an ability to submit a project for code review.
+# Getting started section
+  - In this project,3 files are added,
+
+  - A .html file that includes a header with the text "Hello Microverse!".
+
+  - A .css file that includes one class with styles for your header.
+
+  - A test.md file with the text "This file should be ignored by git".
```
- [ ] Good implementation of changes required, but you missed some point in the `Getting started section`. In this section, you explain how the users can set up your project locally and what users need in their computer  (for example they want a text editor, git installed,...) 🙃 🙃

PraveenKumar141095 replied:
okay thanks but i lost my all 3 review request

cynthiainga reviewed (changes requested):
Hi @Praveen1995-10-14, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![](https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif)

 ## Highlights

- [ ] No linters ✔️ 
- [ ] Correct use of GitHub flow ✔️ 
- [ ] Good commit history ✔️ 

 ## Required Changes ♻️

- [ ] As mentioned by the previous reviewers, and it is a requirement for this project please kindly make sure to add a good Pull Request description. The description is a brief outline of things that you implemented in this pull request. 😉
  - Example of a good PR:
![](https://user-images.githubusercontent.com/87197412/184990532-da778ab4-c789-4136-90a9-629eedd969da.png)

- [ ] Please kindly check the comments under the review 👇 👇 for other required changes 🙃 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

PraveenKumar141095 committed:
update my README.md files

PraveenKumar141095 committed:
update README.md

ha-manel reviewed (changes requested):
# STATUS: REQUIRED CHANGES ♻️

## Hi @Praveen1995-10-14 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  

<img src="https://media0.giphy.com/media/z3piokwf0WPH81MOhu/giphy.gif"/>

## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 

## Required Changes:
- [ ] Kindly click on the edit button of your pull request description and change it to make it more descriptive. Example:

>   - Added linter configurations
>   - Added HTML file
>   - Added some styling in a CSS file

- [ ] Kindly update your README file as the previous reviewers said. In getting started section you can say something like
> - Clone the repo: run `git clone git@github.com:Praveen1995-10-14/hello.git`
> - Open `index.html` in any browser. example: chrome, edge...

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

PraveenKumar141095 committed:
A new branch is created. Features:
-Added an HTML file with website content.
-Added a CSS file that applies the style to the HTML file.
-Added a .gitignore file to filter unwanted files.
-Added a Linter file to check possible bugs on the files.

Fixed:

Code errors on the CSS file.
Pull Request description.
Readme.md (main branch) Getting started content.

ichala reviewed (approved):
Hi @Praveen1995-10-14 ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
![68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966](https://user-images.githubusercontent.com/85113567/184604589-430cf4bc-1d2e-4603-976a-7a48a899ea35.gif)

# To Highlight 💪🏻

- Good Readme✔️
- Good PR title and description ✔️


 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
*************************************
# Optional Suggestions 

- N/A



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
